chesster.js

```diff
@@ -7,6 +7,7 @@ var fuzzy = require('./fuzzy_match.js');
 var spreadsheets = require('./spreadsheets.js');
 var http = require('http');
 var moment = require('moment');
+var players = require("./player.js");
```

Comment:
Yes. More of this please 👍 

We should continue to make minor improvements like this.